_config.yml

```diff
@@ -85,6 +85,8 @@ cloud-integration/google-cloud/mysql-database,
 cloud-integration/azure/connect-to-data-in-the-cloud/cosmosdb,
 getting-started/first-steps-sl,
 getting-started/explore-control-features-sl,
+knowledge-base/kb-htmlplaceholder-enable-scrollbars.md,
```

Comment:
Silverlight doesn't have a Knowledge Base section in the documentation. Consider changing those articles to "How To"s.